src/Presets/stubs/Tailwind/build/webpack.config.preset.js

```diff
@@ -33,11 +33,13 @@ module.exports = {
             {
               loader: 'postcss',
               options: {
-                parser: config.enabled.optimize
+                postcssOptions: {
```

Comment:
This was causing the `build` script to fail when choosing Tailwind as a CSS library in the CLI prompt.

See new [`postcss-loader` config shape](https://webpack.js.org/loaders/postcss-loader/).